skema/text_reading/scala/src/main/scala/org/ml4ai/skema/text_reading/serializer/SkemaJSONSerializer.scala

```diff
@@ -267,7 +267,7 @@ object SkemaJSONSerializer {
       val pairs = for {
         (m: Mention, path: odin.SynPath) <- innermap.toList
         edgeUJson = DirectedGraph.triplesToEdges[String](path.toList).map(_.toUJson)
-      } yield (m.id, edgeUJson)
+      } yield (MentionOps(m).id, edgeUJson)
```

Comment:
This call (MentionOps(m)) on a generic mention can fail with a match exception.  MentionOps from processors does not know about, for example, any `org.ml4ai.skema.text_reading.mentions.CrossSentenceEventMention`.  There must not be any of these in the list.  I believe that this potential problem existed before and is not caused by changes in processors.